DDS.mwc

```diff
@@ -12,6 +12,7 @@ workspace {
     ACE_wrappers
     // For the case where the user clones git repository
     ATCD
+    ACE_TAO
```

Comment:
That's fine.  I'll merge this when I merge something else, I don't want to kick off builds just for this.